web/src/message_list_view.js

```diff
@@ -199,7 +199,7 @@ function get_users_for_recipient_row(message) {
     });
 
     function compare_by_name(a, b) {
-        return a.full_name < b.full_name ? -1 : a.full_name > b.full_name ? 1 : 0;
+        return util.strcmp(a.full_name.toLowerCase(), b.full_name.toLowerCase());
```

Comment:
I think there is no need to use `toLowerCase` here.

Reply:
If we don't use ```toLowerCase```, names will be sorted according to ASCII values of their alphabets. To describe more, I shall use the example given in the issue. for users "Polonius, aaron and Zoe", expected order is "aaron, Polonius, Zoe". If we don't use ```toLowerCase```, it will be "Polonius, Zoe , aaron" because ASCII value of 'P' is less than 'a'.

Reply:
It works without using `toLowerCase` when I am testing it, which I think is due to "Intl.Collator", but I guess we can keep the `lowerCase` for now to make sure it works currently even when browser does not support the ECMAScript Internationalization API Specification.

Reply:
Are there modern browsers that don't support that? I think the answer might be basically no, but someone checking caniuse would be helpful to confirm.